### PR TITLE
GHA - Label Conflicted PRs

### PR DESCRIPTION
**Pull request by bakerboy448**
Title

- [ ] - add conflict label (Markus)

### CODE REVIEW

File: .github/workflows/conflict.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: "Label Conflicts"
+on:
+  push:
```
[the remark this answers is not in the document]
Comment (bakerboy448) [reply .github/workflows/conflict.yml:3]: Label the PRs when develop is pushed, yes.

File: .github/workflows/conflict.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        with:
+          conflict_label_name: "conflict"
+          github_token: ${{ github.token }}
+
+          # --- Optional Inputs ---
+          # To make sure the merge commit exactly matches the branch
+          detect_merge_changes: false # or true to handle as conflicts
+          # By default a comment will be left, adding `conflict_comment: ''` will disable comments
+          # The optional `${author}` will be replaced with the username of the pull request
+          # conflict_comment: |
```
Comment (bakerboy448) [inline .github/workflows/conflict.yml:22]: I'd suggest adding this, but will defer to discussion.

**Comment by Qstick** [comment]
This doesn't really have the same utility as the area labels, this just seems like noise. It's fairly clear when a PR is conflicted

**Comment by bakerboy448** [comment]
clear if one goes and looks at the PR itself unless GH updated something?

**Comment by markus101** [comment]
Is there an advantage to seeing that a PR is conflicted from the PR list?

I could actually see some value in notifying the author if they don't address the conflict after X period of time, but I wouldn't want it to go out immediately after the conflict arises.

**Comment by markus101** [comment]
Not something we're going to implement at this time.

**Pull request closed without merging**